migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -0,0 +1,258 @@
+// Module included in the following dir:
+//
+// * service-mesh-docs-main/checklist.adoc
+
+:_mod-docs-content-type: ASSEMBLY
+[id=ossm-migrating-premigration-checklists-assembly]
+= Premigration checklists
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-premigration-checklists-assembly
+
+toc::[]
+
+The checklists make the following assumptions:
+
+1. You have read "Important information to know before migrating" and understand the differences between {SMProduct} 2 and {SMProduct} 3.
+2. You have read through the migration recommendations.
+3. You want to migrate from {SMProduct} 2 to {SMProduct} 3.
+4. You are running {SMProduct} {SMv2Version}.
+5. You have upgraded your `ServiceMeshControlPlane` to the latest version.
+6. If you are using {KialiProduct}, you are running the latest version.
+7. You have installed the {SMProduct} Operator 3.
+
+To install the {SMProduct} 3 Operator, see: xref:../../install/ossm-installing-openshift-service-mesh.adoc[Installing OpenShift Service Mesh]
+
+[WARNING]
+====
+You must upgrade your {SMProduct} Operator 2 to the latest release **before** you install {SMProduct} Operator 3.
+
+If you upgrade {SMProduct} Operator 2 operator _after_ you install {SMProduct} Operator 3, you will need to uninstall and reinstall {SMProduct} Operator 3 to ensure the included custom resource definitions (CRDs) are up-to-date.
+====
+//Content is taken from here: https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+
+//check for updates at https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/README.md
+
+== Checklist for configuring add on replacements in Service Mesh 2
+
+After you have installed {SMProduct} Operator 3, you need to disable and configure a replacement another option for the following add ons in your `ServiceMeshControlPlane` in {SMProduct} 2:
```

Comment:
"and configure a replacement another option for the following add ons" 
It seems to be a bit confusing